rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -718,8 +745,14 @@ boolean forget() {
                     forgetURI, e.getMessage());
                 // TODO write a test to ensure that recovery only retries the forget request
                 return false; // force recovery to keep retrying
+            } catch (Exception e) {
```

Comment:
Similar comment as above. We are using JAX-RS so we need to use the WebApplicationException since that will already hold the relevant status code.

Reply:
here the WebApplicationException is caught in the previous catch block.

Reply:
The WebTarget, Invocation.Builder, AsyncInvoker and get on Future<Response> methods we are using all ultimately return a Response object to the caller so you don't need to catch checked exceptions.

Reply:
I will leave this thread unresolved since you don't need to catch checked exceptions.